specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json

```diff
@@ -235,14 +235,14 @@
           }
         ],
         "responses": {
-          "201": {
-            "description": "Namespace create request accepted.",
+          "200": {
```

Comment:
>200 [](start = 11, length = 3)

Did the response code change in the backend or are we just making swagger match reality?

Reply:
I think it is just an order change rather than actual status code change due to the swagger tool that I used. it is just swapping line 238 with 244 apparently

Reply:
Yes, you're right. I missed it.

---
In reply to: [400542968](https://github.com/Azure/azure-rest-api-specs/pull/8640#discussion_r400542968) [](ancestors = 400542968)